test/stdlib/TestData.swift

```diff
@@ -1215,74 +1287,2669 @@ class TestData : TestDataSuper {
-DataTests.test("test_classForCoder") { TestData().test_classForCoder() }
-DataTests.test("test_AnyHashableContainingData") { TestData().test_AnyHashableContainingData() }
-DataTests.test("test_AnyHashableCreatedFromNSData") { TestData().test_AnyHashableCreatedFromNSData() }
-DataTests.test("test_noCopyBehavior") { TestData().test_noCopyBehavior() }
-DataTests.test("test_doubleDeallocation") { TestData().test_doubleDeallocation() }
-DataTests.test("test_repeatingValueInitialization") { TestData().test_repeatingValueInitialization() }
-DataTests.test("test_rangeZoo") { TestData().test_rangeZoo() }
-DataTests.test("test_sliceAppending") { TestData().test_sliceAppending() }
-DataTests.test("test_replaceSubrange") { TestData().test_replaceSubrange() }
-DataTests.test("test_sliceWithUnsafeBytes") { TestData().test_sliceWithUnsafeBytes() }
-DataTests.test("test_sliceIteration") { TestData().test_sliceIteration() }
-DataTests.test("test_unconditionallyBridgeFromObjectiveC") { TestData().test_unconditionallyBridgeFromObjectiveC() }
-DataTests.test("test_sliceIndexing") { TestData().test_sliceIndexing() }
-DataTests.test("test_sliceEquality") { TestData().test_sliceEquality() }
-DataTests.test("test_sliceEquality2") { TestData().test_sliceEquality2() }
-DataTests.test("test_splittingHttp") { TestData().test_splittingHttp() }
-DataTests.test("test_map") { TestData().test_map() }
-DataTests.test("test_dropFirst") { TestData().test_dropFirst() }
-DataTests.test("test_dropFirst2") { TestData().test_dropFirst2() }
-DataTests.test("test_copyBytes1") { TestData().test_copyBytes1() }
-DataTests.test("test_copyBytes2") { TestData().test_copyBytes2() }
-DataTests.test("test_sliceOfSliceViaRangeExpression") { TestData().test_sliceOfSliceViaRangeExpression() }
-DataTests.test("test_appendingSlices") { TestData().test_appendingSlices() }
-DataTests.test("test_sequenceInitializers") { TestData().test_sequenceInitializers() }
-DataTests.test("test_reversedDataInit") { TestData().test_reversedDataInit() }
-DataTests.test("test_replaceSubrangeReferencingMutable") { TestData().test_replaceSubrangeReferencingMutable() }
-DataTests.test("test_replaceSubrangeReferencingImmutable") { TestData().test_replaceSubrangeReferencingImmutable() }
-DataTests.test("test_replaceSubrangeFromBridged") { TestData().test_replaceSubrangeFromBridged() }
 
+        func test_validateMutation_withUnsafeMutableBytes() {
+        var data = Data(bytes: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
+        data.withUnsafeMutableBytes { (ptr: UnsafeMutablePointer<UInt8>) in
+            ptr.advanced(by: 5).pointee = 0xFF
+        }
+        expectEqual(data, Data(bytes: [0, 1, 2, 3, 4, 0xFF, 6, 7, 8, 9]))
+    }
+    
+    func test_validateMutation_appendBytes() {
+        var data = Data(bytes: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
+        data.append("hello", count: 5)
+        expectEqual(data[data.startIndex.advanced(by: 5)], 0x5)
```

Comment:
This test seems odd since it doesn't test that it actually appended the new bytes.